doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -301,8 +301,16 @@ set with::
 SPDK Usage
 ==================
 
-If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
-number here with "spdk:" prefix for ``bluestore_block_path``.
+If you want to use SPDK driver for NVME SSD, at the first some hugepages must
```

Comment:
s/at the first/To start with,/

i'd suggest use the active voice when writing document. i.e.

```rst
If you want to use SPDK driver for NVME SSD, you need to ready your system for it. Please refer to `SPDK document`_ for more details.

.. SPDK document: http://www.spdk.io/doc/getting_started.html#getting_started_examples
```